src/localPipResolver.js

```
@@ -14,7 +14,10 @@ function LocalPipService(datapath, layers) {
   const self = this;
 
   createPipService(datapath, _.defaultTo(layers, []), false, (err, service) => {
-     self.pipService = service;
+    if (err) {
```

wait we were swallowing all errors here? hah. well, good thing we fixed it.